modules/custom/az_block_types/az_custom_menu_block/az_custom_menu_block.module

```diff
@@ -31,3 +31,30 @@ function az_custom_menu_block_form_block_content_az_custom_menu_block_form_alter
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function az_custom_menu_block_preprocess_block(&$variables) {
```

Comment:
Should possibly narrow scope to the sidebar template.

Reply:
Seems like a good idea if possible.